src/runtime/virtcontainers/hypervisor.go

```diff
@@ -818,6 +818,7 @@ type hypervisor interface {
 	// getPids returns a slice of hypervisor related process ids.
 	// The hypervisor pid must be put at index 0.
 	getPids() []int
+	getVirtioFsPid() *int
```

Comment:
ooi, does this need to return a pointer?

Reply:
It could be zero as well if there is not virtiofsd process, I decided to use a pointer, if NULL this is  there is not process to monitor